api/http/api_inventory.go

```diff
@@ -51,6 +51,7 @@ const (
 	queryParamSort           = "sort"
 	queryParamHasGroup       = "has_group"
 	queryParamValueSeparator = ":"
+	queryParamScopeSeparator = "/"
```

Comment:
originally it was `:` I think. 

I don't mind `/` but I'm worried about the encoding. I can't get a straight answer out of the RFC whether it's a reserved char or not: https://tools.ietf.org/html/rfc3986#section-3.4.

Reply:
@mchalski it is not and can be safely used